handlers/main.yml

```diff
@@ -1,2 +1,5 @@
 ---
-# handlers file for ansible-role-nifi-registry
\ No newline at end of file
+# handlers file for ansible-role-nifi-registry
+
+- name: reload systemctl
```

Comment:
you can use systemd module here: https://docs.ansible.com/ansible/latest/modules/systemd_module.html

Reply:
@raphapr adjusted